tests/core/gui/test_revoked_user.py

```diff
@@ -36,21 +36,28 @@ async def test_revoked_notification(
     ).dump_and_sign(alice_user_fs.device.signing_key)
     central_widget = logged_gui.test_get_central_widget()
     assert central_widget is not None
-    async with aqtbot.wait_signal(central_widget.new_notification):
+    async with aqtbot.wait_signal(central_widget.new_notification, timeout=3000):
```

Comment:
default timeout is 1s, why is this not enough ?

Reply:
1second is generating random timeout.

Reply:
can't we avoid this by using `aqtbot.wait_until(callbable)` instead of `wait_signal` ? (`wait_until` is much more reliable)